### ISSUE TITLE
tanzu package fails with Error: no matches for kind "PackageMetadata" in version "data.packaging.carvel.dev/v1alpha1"

### ISSUE THREAD

**Issue by ganga-sah** [vmware-tanzu/community-edition#2166]
## Bug Report

I am getting error: Error: no matches for kind "PackageMetadata" in version "data.packaging.carvel.dev/v1alpha1" as in command "tanzu package available list -A" and output below. I see same error from any other "tanzu package ..." command. 

✖  exit status 1 

## Expected Behavior

I should not see error from above command.

## Steps to Reproduce the Bug
Run command below in v0.9.1:
 tanzu package available list -A

## Screenshots or additional information and context

I get error from any "tanzu package ..." command as in command line output below:

$ tanzu package available list -A
- Retrieving available packages... Error: no matches for kind "PackageMetadata" in version "data.packaging.carvel.dev/v1alpha1"
Usage:
  tanzu package available list or list PACKAGE_NAME [flags]

Examples:
  
    # List available packages across all namespaces 	
    tanzu package available list -A
	
    # List all versions for available package from specified namespace	
    tanzu package available list contour.tanzu.vmware.com --namespace test-ns

Flags:
  -h, --help   help for list

Global Flags:
  -A, --all-namespaces      If present, list packages across all namespaces, optional
      --kubeconfig string   The path to the kubeconfig file, optional
      --log-file string     Log file path
  -n, --namespace string    Namespace of packages, optional (default "default")
  -o, --output string       Output format (yaml|json|table), optional
      --verbose int32       Number for the log level verbosity(0-9)

Error: exit status 1

## Environment Details

* Build version (`tanzu version`): 
My version is as below:
$ tanzu version  
version: v0.2.1
buildDate: 2021-09-29
sha: ceaa474

* Deployment (Managed/Standalone cluster): 
Standalone

* Infrastructure Provider (Docker/AWS/Azure/vSphere): 
Docker 

* Operating System (client):
MacOS Big Sur 11.3.1

## Diagnostics and log bundle
See full error/output above.

**Comment by joshrosso** [comment]
Hi @ganga-sah!

This should only happen if `kapp-controller` does not start-up successfully. Your cluster thinks the CRD is missing and it is installed as part of the `kapp-controller` deployment.

Would you be able to re-bootstrap the cluster and give us the output of:

```
kubectl get po -A
```

If you see `kapp-controller` started up, then this command *should* work. You may also want to make sure your kube context is pointed to the correct cluster (`kube config get-contexts`) before running the command.

Lastly, if you'd like us to help out in real-time, feel free to hit us up in #tanzu-community-edition (k8s slack)

**Comment by ganga-sah** [comment]
Hi @joshrosso Thanks for your reply!
After above issue, I had uninstalled my setup. Today I tried the same installation steps again as per the guide using the same version (0.9.1) but this time "tanzu package available list -A" returned following output instead of above error, so I am good now and we can close this issue.

/ Retrieving available packages... 
  NAME                                                DISPLAY-NAME                       SHORT-DESCRIPTION                                                                                                                                                                                       NAMESPACE   
  addons-manager.tanzu.vmware.com                     tanzu-addons-manager               This package provides TKG addons lifecycle management capabilities.                                                                                                                                     tkg-system  
  ako-operator.tanzu.vmware.com                       ako-operator                       NSX Advanced Load Balancer using ako-operator                                                                                                                                                           tkg-system  
  antrea.tanzu.vmware.com                             antrea                             networking and network security solution for containers                                                                                                                                                 tkg-system  
  calico.tanzu.vmware.com                             calico                             Networking and network security solution for containers.                                                                                                                                                tkg-system  
  kapp-controller.tanzu.vmware.com                    kapp-controller                    Kubernetes package manager                                                                                                                                                                              tkg-system  
  load-balancer-and-ingress-service.tanzu.vmware.com  load-balancer-and-ingress-service  Provides L4+L7 load balancing for TKG clusters running on vSphere                                                                                                                                       tkg-system  
  metrics-server.tanzu.vmware.com                     metrics-server                     Metrics Server is a scalable, efficient source of container resource metrics for Kubernetes built-in autoscaling pipelines.                                                                             tkg-system  
  pinniped.tanzu.vmware.com                           pinniped                           Pinniped provides identity services to Kubernetes.                                                                                                                                                      tkg-system  
  vsphere-cpi.tanzu.vmware.com                        vsphere-cpi                        The Cluster API brings declarative, Kubernetes-style APIs to cluster creation, configuration and management. Cluster API Provider for vSphere is a concrete implementation of Cluster API for vSphere.  tkg-system  
  vsphere-csi.tanzu.vmware.com                        vsphere-csi                        vSphere CSI provider                                                                                                                                                                                    tkg-system  


Thank you for you support!
